### ISSUE TITLE
Messages are not in order

### ISSUE THREAD

**Issue by acagastya** [FruitieX/teleirc#191]
![screenshot from 2017-06-28 23 15 09](https://user-images.githubusercontent.com/5466083/27652104-1cece1a4-5c58-11e7-8d1a-f9f746aa1113.png)
![photo_2017-06-28_23-19-04](https://user-images.githubusercontent.com/5466083/27652129-3ce30880-5c58-11e7-8cda-c5f7851d129c.jpg)

The messages relayed to Telegram are out of order.

**Comment by Mikaela** [comment]
Can you see them in the correct order with debug logging (I think that is `teleirc -vvv`)? I wonder if it's the bot receiving the messages in wrong order from the IRC server or if the bot is doing some processing afterwards causing this issue.

Disclaimer: I am not a coder and I am just trying to think what they would ask.

**Comment by acagastya** [comment]
I can't see what had happened then -- the application was running on my laptop, not on a virtual server.

**Comment by FruitieX** [comment]
I wonder if it's just the Telegram servers that are handing you the messages in this order, since they are so close to each other in time.

**Comment by acagastya** [comment]
No, that would not be the case since each telegram messages has a unique ID so a message would not be sent  until the previous one is sent. 

And in this case, gpy is a bot which changes links in wiki markup and reply with the URL.

**Comment by Mikaela** [comment]
I have started noticing this in multiple groups with separate bots and I am told that IRC server cannot send messages in wrong order either, so I am setting label bug and commenting that it's not just you.

I will be keeping the feedback-required label until I or someone affected gets logs.

**Comment by acagastya** [comment]
![screenshot from 2017-09-11 19-26-55](https://user-images.githubusercontent.com/5466083/30278319-b3882240-9727-11e7-9e38-2368b9b7d995.png)
![screenshot from 2017-09-11 19-27-35](https://user-images.githubusercontent.com/5466083/30278310-aefd6190-9727-11e7-9bab-0e4262083f10.png)

It is still happening.
